src/const.ts

```diff
@@ -53,12 +54,13 @@ export const DEFAULT_CONSTANT: Constant = {
     width: '16px',
     height: '16px',
     onClick: noop,
+    style: {},
```

Comment:
where is this from?

Reply:
It just default value

Reply:
i meant there is no property called style in `Constant`

Reply:
It's a `Constant.customRefreshComponent.style`

Reply:
oh my bad!!